pallets/nfts/src/tests/spec.rs

```diff
@@ -21,7 +22,8 @@ fn create_register_details() {
         let mock_details = MockNFTDetails::WithU8(42);
         assert_ok!(NFTs::create(
             RawOrigin::Signed(ALICE).into(),
-            mock_details.clone()
+            mock_details.clone(),
+            None
```

Comment:
We need to test the stuff with some `series_details` too

Reply:
> That's true. If didn't want to fully fix the tests or the benchmark code until I got a approval that my approach makes sense :D